site/themes/multi_course/layouts/shortcodes/getpage.html

```diff
@@ -0,0 +1,14 @@
+{{- if in (.Get 0) "/_index.md" -}}
```

Comment:
I think we can take the contents of this file (`/site/themes/multi_course/layouts/shortcodes/getpage.html`) and place it in `/site/layouts/shortcodes`.  Since we're moving to the course page being a Hugo theme itself, layouts placed in the `layouts` folder of a project inheriting the theme should override the layouts in the theme itself.  We can also remove the `single_course` theme.

Reply:
(we could also leave this for a follow-up PR)

Reply:
should I just remove both the `multi_course` and `single_course` themes entirely?

Reply:
I went for that and just copied the `home.html` in `site/layout`

Reply:
👍